dataland-community-manager/src/main/kotlin/org/dataland/datalandcommunitymanager/services/EmailBuilder.kt

```diff
@@ -0,0 +1,155 @@
+package org.dataland.datalandcommunitymanager.services
+
+import org.dataland.datalandcommunitymanager.model.dataRequest.BulkDataRequest
```

Comment:
The user is only informed about accepted identifiers, correct?
Would it have merit to also inform about the rejected ones?

Reply:
The user is informed about accepted and rejected identifiers (in the response of the POST request)

However, the receivers of the notification mail are only informed about accepted ones.
At first, also the rejected ones were included in the mail, but Siegfried said that the rejected ones do not really matter to recipients of that mail (Erik and dataland dfine inbox).  So we removed them from the mail.

Reply:
Ok makes sense to me